changelog.d/+kafka-settings-from-secret.internal.md

```diff
@@ -0,0 +1 @@
+Add optional load_from_secret field to Kafka client config spec to allow setting properties from a Secret.
```

Comment:
```suggestion
Add optional `loadFromSecret` field to Kafka client config spec to allow setting properties from a Secret.
```
We rename fields to camelCase, I think it's better to use the name from resource definition (changelog is for the users)